account_payment_order/models/account_move_line.py

```diff
@@ -42,6 +42,8 @@ def _compute_partner_bank_id(self):
                 )
             ):
                 ml.partner_bank_id = ml.move_id.invoice_partner_bank_id.id
+            else:
+                ml.partner_bank_id = False
```

Comment:
Being stored=True, it shouldn't be needed. But anyway, it's better to assign the same previous value:
```suggestion
                ml.partner_bank_id = ml.partner_bank_id
```

Reply:
Maybe the problem is the absence of the default?